### PR TITLE
Add direct link to tyk cloud payment method (add-payment-method.md)

### PR DESCRIPTION
**Pull request by letzya**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

1. Added direct link to tyk cloud so users don't need to look up the screen on their own.
2. While doing (1) I found that Hugo double-escape special chars so I used raw HTML instead. 
3. The external rendering did not work (it opens external links in a new tab and adds an external icon next to the text) - so I changed it to match anything that is not `tyk.io/docs` instead of just `tyk.io` 
4. Improve the content:
   1.  what happens if you are not a billing admin 
   2. Reworded the NOTE message for tyk cloud payment support

Testing:
1. Login to Tyk cloud as "Billing Admin" user
3. Click on the link in "ACCOUNT & BILLING –> Payment Method" (https://deploy-preview-3835--tyk-docs.netlify.app/docs/nightly/tyk-cloud/account-billing/add-payment-method/) and see that 
   1. It opens in another tab
   4. It opens on the correct screen

 
---

### Preview Link
https://deploy-preview-3835--tyk-docs.netlify.app/docs/nightly/tyk-cloud/account-billing/add-payment-method/

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

**Comment by letzya** [comment]
@SonjaChevre can you please approve the content?

**Review by yurisasuke — APPROVED** [review]

File: tyk-docs/content/tyk-cloud/account-billing/add-payment-method.md [hunk 1]
```diff
@@ -1,38 +1,31 @@
...
+**Note:** You must have *Billing Admin* user rights to add a payment method. 
 
-You must have billing admin permissions to add a payment method. 
-
-1. Log in to your account as a Billing Admin.
-2. If you have dashboard and billing permissions, click the Account & Billing menu item (otherwise you will automatically log into the Account and Billing).
-3. Click the Payment Method menu item.
+Follow these steps to add a payment method:
+1. Make sure you are logged in to your *Tyk Cloud* UI as a *Billing Admin* user.
+2. Go to <a href="https://account.cloud-ara.tyk.io/payment-method" class="external-links" target="_blank" rel="noopener">ACCOUNT & BILLING --> Payment Method</a> menu item. In case you don't have this user rights, you will automatically logged into the main [OPRATIONS](https://dashboard.cloud-ara.tyk.io/) screen (the main login page).
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-cloud/account-billing/add-payment-method.md]:
```suggestion
2. Go to <a href="https://account.cloud-ara.tyk.io/payment-method" class="external-links" target="_blank" rel="noopener">ACCOUNT & BILLING --> Payment Method</a> menu item. If you are not a *Billing Admin* user, you will be automatically logged into the main [OPERATIONS](https://dashboard.cloud-ara.tyk.io/) screen (the main login page).
```

File: tyk-docs/content/tyk-cloud/account-billing/add-payment-method.md [hunk 2]
```diff
@@ -1,38 +1,31 @@
...
+3. Enter your card details and click *Save*.
+4. You'll see confirmation that the payment method was added successfully.
 
 {{< note success >}}
-**Note**
+**Note about card payments**
   
-For this release of Tyk Cloud, we are only supporting card payments
+Currently, *Tyk Cloud* only supports card payments. If you requeire another payment method, please [constact us](https://tyk.io/contact/) to speak with a member of our team.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-cloud/account-billing/add-payment-method.md]:
```suggestion
Currently, *Tyk Cloud* only supports card payments. If you require another payment method, please [contact us](https://tyk.io/contact/) to speak with a member of our team.
```

**Comment by letzya** [comment]
@SonjaChevre from your perspective, is there any reason not to merge?

**Comment by SonjaChevre** [comment]
I have spotted a couple of things that need to be changed. Will try to review today or tomorrow latest.

**Review by SonjaChevre — APPROVED** [review]
good from my side - have taken the opportunity to improved the wording on that page slightly.

One question from my side: do we have a guide for the documentation that says how we format navigation item from the UI? 

Here we use:
[ACCOUNT & BILLING –> Payment Method](https://account.cloud-ara.tyk.io/payment-method)

The first navigation level is in capital, the second level is not. In some other parts of the documentation I have seen pages where we use bold or italic.

**Review by dcs3spp — COMMENTED** [review]
Adding comment via GHub web interface, so I can commit minor suggestions

**Review by dcs3spp — APPROVED** [review]
PR LGTM

**Pull request merged**